app/services/parameter_safe_render.rb

```diff
@@ -6,18 +6,36 @@ def initialize(host)
     @host = host
   end
```

Comment:
Would it make sense to include 
`
@interpolate = Setting[:interpolate_erb_in_parameters]
`
in the initializer so it is only looked up once (assuming the same renderer is reused) and not on every render?

Reply:
Good idea, one renderer is actually shared but only N times (constant). Won't hurt.